src/main/resources/assets/magic_crystal/lang/zh_cn.json

```diff
@@ -0,0 +1,10 @@
+{
+  "item.magic_crystal.hammer": "锤子",
+  "item.magic_crystal.raw_tin": "粗锡",
+  "item.magic_crystal.iron_plate": "铁板",
+  "item.magic_crystal.rubber_bowl": "一碗橡胶",
```

Comment:
Should we separate rubber and latex?

Reply:
There will be a "synthetic rubber" item.